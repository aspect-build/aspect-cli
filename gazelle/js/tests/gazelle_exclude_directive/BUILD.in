# gazelle:generation_mode update

# Simple
# gazelle:exclude **/*.ig.ts

# Should support sub-directories
# gazelle:exclude subdir/ignored.ts
# gazelle:exclude **/sub-ignored.ts

# Sub-directories must be relative to BUILD
# gazelle:exclude sub-not-ignored.ts

# Should not be able to exclude for a different BUILD
# gazelle:exclude sub-ignored.ts
# gazelle:exclude subproject/*
# gazelle:exclude subproject-included/*
