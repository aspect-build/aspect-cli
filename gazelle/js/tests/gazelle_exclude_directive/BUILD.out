load("@aspect_rules_ts//ts:defs.bzl", "ts_project")

# gazelle:generation_mode update

# Simple
# gazelle:exclude **/*.ig.ts

# Should support sub-directories
# gazelle:exclude subdir/ignored.ts
# gazelle:exclude **/sub-ignored.ts

# Sub-directories must be relative to BUILD
# gazelle:exclude sub-not-ignored.ts

# Should not be able to exclude for a different BUILD
# gazelle:exclude sub-ignored.ts
# gazelle:exclude subproject/*
# gazelle:exclude subproject-included/*

ts_project(
    name = "gazelle_exclude_directive",
    srcs = [
        "main.ts",
        "subdir-included/included-lib.ts",
        "subdir/sub-not-ignored.ts",
    ],
)
